src/api/routes/offer.js

```diff
@@ -150,7 +150,10 @@ module.exports = (app) => {
         validators.canDisable,
         async (req, res, next) => {
             try {
-                const offer = await (new OfferService())
+                const offerService = new OfferService();
+
+                await offerService.sendOfferDisabledNotification(req.params.offerId);
```

Comment:
You should only do this once the offer is actually disabled, i.e., in case the next call fails, the email would still be sent, which is bad. You can place the email sending line after the disable call, and if the first one throws, the email won't be sent